js/scripts.js

```diff
@@ -237,6 +237,10 @@ function placeMarkers(institutions) {
         
         // Center 2 digit-long student counters
         let x = numStudents >= 10 ? 2 : 26;
+        // Check if the user is on a mobile device, and adjust svgs accoringly
+        if (/Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(navigator.userAgent)){
+            var mobile = true;
+        }
```

Comment:
This is one way to check for mobile, but you could also do a css media query or a js media query: https://developer.mozilla.org/en-US/docs/Web/API/Window/matchMedia.

I think this is fine though.

I might suggest moving `mobile` outside the `if`, defaulting it to false, and setting true in the if though. Up to you.